pkg/version/version.go

```diff
@@ -0,0 +1,13 @@
+package version
+
+import "fmt"
+
+var (
+	GitCommit string
+	BuildTime string
+	Version   string
+)
+
+func String() string {
+	return fmt.Sprintf("%s, Commit:%s, Build-time:%s", Version, GitCommit, BuildTime)
```

Comment:
I think you could use os.ARGS[0] to get the binary name and then not need to add alerter/collector/ingestor in the other spots.